totalRP3/Modules/Register/Characters/PlayerMapScan/PlayerMapPin.lua

```diff
@@ -59,15 +59,15 @@ function TRP3_PlayerMapPinMixin:GetDisplayDataFromPoiInfo(poiInfo)
 		displayData.categoryPriority = -1;
 	else
 		local relation = player:GetRelationshipWithPlayer()
-		if relation ~= TRP3_API.register.relation.NONE then
+		if relation.id ~= TRP3_API.register.relation.getRelationInfo('NONE').id then
```

Comment:
Assuming 'NONE' is a magic string here I'd _prefer_ it get a proper constant somewhere; need to think some more on it.

Reply:
Agree absolutely, i can think of a handful of fixes, i'll take another look at this.